src/Datasource/Paging/NumericPaginator.php

```diff
@@ -470,7 +470,7 @@ protected function mergeOptions(array $params, array $settings): array
     {
         if (!empty($settings['scope'])) {
             $scope = $settings['scope'];
-            $params = !empty($params[$scope]) ? (array)$params[$scope] : [];
+            $params = empty($params[$scope]) ? [] : (array)$params[$scope];
```

Comment:
Can this be modified like this?
```suggestion
            $params = (array)$params[$scope] ?? [];
```

Reply:
I would not get rid of empty() for the sake of it. It is fine being used :)

Reply:
Do you want to keep the changes as they are or ignore this rule or change is somehow?

Reply:
I am for ignore, since it reduces readability.